enterprise/backend/src/metabase_enterprise/data_editing/foreign_keys.clj

```diff
@@ -0,0 +1,120 @@
+(ns metabase-enterprise.data-editing.foreign-keys
+  (:require
+   [clojure.java.jdbc :as jdbc]))
+
+(defn- reducible-children-thunks
```

Comment:
Unused, pending us storing the required table metadata, which we will also need sync to start populating.

It will also need to be written in terms of driver methods, eventually, although we could cheat for now while we're only supporting mainstream RDBMs.